backend/BusinessLogic/Services/VideoService/VideoService.cs

```diff
@@ -145,5 +146,21 @@ private static void MergeChunks(string chunk1, string chunk2)
                 File.Delete(chunk2);
             }
         }
+
+        public async Task<List<Video>> GetUserVideos(Guid userId)
+        {
+            var videos = await _videosRepository.GetAllVideos();
```

Comment:
Seems like a big waste to GetAllVideos (of which there could be hundreds) and then filter them. Could you maybe implement a method in the videos repository to get user's videos? This method would be a proxy then

Reply:
I will. And I will make another one for deleted videos